src/main/java/com/example/onlinebookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.onlinebookstore.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
What is the reason to make this exception checked?

Reply:
Actually, this is specified by the requirements for this task:
> Add AuthenticationController with the following methods:
public UserResponseDto register(UserRegistrationRequestDto request) throws RegistrationException